src/pug/react/autocomplete.pug

```diff
@@ -0,0 +1,19 @@
+extends ../_docs-template
+
+block vars
+  - var activeLink = 'react';
+  - var title = 'Autocomplete React Component | Framework7 React Documentation';
+
+block docs-index
+
+block content
+  .docs-nav
+    include ../_docs-menu-react
+  .docs-content
+    .motion-docs-page-title
+        h1 Autocomplete React Component
+        script(src="/js/motion/autocomplete.js")
```

Comment:
All scripts with motion components must have an `async` attribute. Look how it is done in other pages